RealmSwift/App.swift

```diff
@@ -109,3 +109,122 @@ extension App {
         self.__setASAuthorizationControllerDelegateFor(controller)
     }
 }
+
+#if canImport(Combine)
+import Combine
+
+@available(OSX 10.15, watchOS 6.0, iOS 13.0, iOSApplicationExtension 13.0, OSXApplicationExtension 10.15, tvOS 13.0, macCatalyst 13.0, macCatalystApplicationExtension 13.0, *)
+extension EmailPasswordAuth {
```

Comment:
All of these methods need documentation for their return value. The extension should also be `public` (though I believe having all public methods suffices this, I personally find it clearer when an extension is public).

Reply:
Added documentation, changed visibility